src/Haskell/Ide/Engine/Version.hs

```diff
@@ -1,93 +1,26 @@
-{-# LANGUAGE CPP                   #-}
-{-# LANGUAGE DataKinds             #-}
-{-# LANGUAGE GADTs                 #-}
-{-# LANGUAGE OverloadedStrings     #-}
-{-# LANGUAGE PartialTypeSignatures #-}
-{-# LANGUAGE TemplateHaskell       #-}
-module Haskell.Ide.Engine.Plugin.Base where
+{-# LANGUAGE CPP, TemplateHaskell, OverloadedStrings #-}
+-- | Information and display strings for HIE's version
+-- and the current project's version
+module Haskell.Ide.Engine.Version where
 
 import           Control.Exception
-import           Data.Aeson
-import           Data.Foldable
-import qualified Data.Map                        as Map
 import           Data.Maybe
-import qualified Data.Text                       as T
-import qualified Data.Versions                   as V
 import           Development.GitRev              (gitCommitCount)
 import           Distribution.System             (buildArch)
 import           Distribution.Text               (display)
-import           Haskell.Ide.Engine.MonadTypes
+import           Options.Applicative.Simple      (simpleVersion)
 import           Haskell.Ide.Engine.Cradle (isStackCradle)
 import qualified HIE.Bios.Types as BIOS
-import           Options.Applicative.Simple      (simpleVersion)
 import qualified Paths_haskell_ide_engine        as Meta
-
+import qualified System.Log.Logger as L
+import qualified Data.Text                       as T
+import qualified Data.Versions                   as V
 import           System.Directory
 import           System.Info
 import           System.Process
-import qualified System.Log.Logger as L
-
--- ---------------------------------------------------------------------
-
-baseDescriptor :: PluginId -> PluginDescriptor
```

Comment:
Same question as for hoogle plugin. What does this entail for lsp-clients?

Reply:
This has no change for LSP clients, it was only used for querying information about plugins from the JSON transport and isn't accessible via LSP